lib/Doctrine/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -161,6 +162,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $writer->write($path, $direction, $sql);
         }
 
+        $this->io->success(sprintf(
+            'Successfully migrated version(s) : %s : [%s]',
+            implode(', ', $versions),
+            strtoupper($plan->getDirection())
+        ));
```

Comment:
Could be cool to have one message format for singular, and another with a list for plural:

```
Successfully migrated versions [UP]:
    - My\Migration1
    - My\Migration2
    - My\Migration3
```

Reply:
No answer… well, let's merge this I guess.